src/reducer.js

```diff
@@ -13,6 +14,12 @@ const getJSONResponse = action => action.payload.json;
 const getError = action => action.payload.error;
 const getPreviousError = (state, action) => state[getName(action)] ? state[getName(action)].error : null;
 
+const includeString = (element, array) => array.indexOf(element) !== -1;
+const resetOrKeepValue = (field, action, currentData) => (
+  includeString(field, action.payload.data) ? undefined : currentData[field]
```

Comment:
similar with `false` or `0`. I think in normalizeData function, let check `!Array.isArray(data)` instead